app/src/main/java/org/gnucash/android/export/Exporter.java

```diff
@@ -53,7 +53,7 @@
  * @author Ngewi Fet <ngewif@gmail.com>
  * @author Yongxin Wang <fefe.wyx@gmail.com>
  */
-public abstract class Exporter implements Closeable {
+public abstract class Exporter {
```

Comment:
why not Closeable ?

Reply:
As I mentioned in the commit description, the subclasses of Exporter already call close() in generateExport(), so there is no need for the users of the class to call close().